src/main/java/com/xebialabs/restito/semantics/Action.java

```diff
@@ -293,6 +295,33 @@ public Response apply(Response input) {
     }
 
     /**
+     * Creates a sequence action which contains all passed actions and
+     * executes one by one of them in the same order if {@link Action#apply(Response)} is repeated.
+     * If all passed actions has been already applied it behaves like {@link #noop()} action.
```

Comment:
You are right it can be confusing. Do you have any preferred way how to configure it?